terraform.tf

```
@@ -1,8 +1,11 @@
 terraform {
   required_providers {
-    azurerm = {
-      source  = "hashicorp/azurerm"
-      version = ">= 3.7.0"
+    azapi = {
+      source = "Azure/azapi"
```

```suggestion
      source = "Azure/azapi"
      version = ">= 1.0.0"
```
We should at least exclude preview versions.